app/jobs/send_texts_job.rb

```diff
@@ -0,0 +1,14 @@
+class SendTextsJob < ApplicationJob
+  queue_as :default
+
+    def perform   
+        User.all.each do |user|
+          location = "#{user.lat},#{user.long}"
```

Comment:
You've used this same line previously, could #location be an attribute on the User table so you could just call User.location?